x/auth/client/cli/tx.go

```diff
@@ -17,8 +17,7 @@ import (
 )
 
 // GetTxCmd returns the transaction commands for this module.
- // NOTE: Auth tx commands from Cosmos are mounted directly under the root.
- func GetTxCmd(cdc *codec.Codec) *cobra.Command {
+func GetTxCmd(cdc *codec.Codec) *cobra.Command {
 	txCmd := &cobra.Command{
```

Comment:
could we replace this with original cosmos GetTxCmd? and then we can substitute the routing to here